docs/editor/intellisense.md

```diff
@@ -0,0 +1,119 @@
+
+In other tools, IntelliSense goes by the terms "code completion", "auto code completion", "content assist", or "code hinting." The IntelliSense experience in VS Code is best-in-class, providing the most intelligent, context-aware suggestions possible. 
+
+## Using IntelliSense
+
+IntelliSense is easy to use. Simply begin typing. While you type the language service will provide code completion and on certain characters (for example `.`) method suggestions will appear. 
+
+> Tip: You can trigger IntelliSense in any editor window by typing <kbd>ctrl+space</kbd>.
+
+In addition, you can see the short documentation (commonly called **quick outline**) for each method (as provided by the language service). 
+
+![quick outline](images/intellisense/quick_outline.png)
+
+After choosing a method you are provided with **paramater info**. 
+
+![paramater info](images/intellisense/paramater_info.png)
+
+## Suggestion Priority
+
+IntelliSense is context aware and will provide you with the best suggestions possible. The order and the icon to the left of the word indicate the priority of each suggestion. 
+
+> **Higher suggestion priority = higher order and a more detailed icon**
+
+This image shows the result of triggering IntelliSense on `express` a popular [Node.js module](https://expressjs.com/). 
+
+![image showing intellisense icons](images/intellisense/intellisense_icons.png)
+
+Methods, variables, and objects tend to have high priority. Simple word completion, tends to be lower priority. When the priority level is equivalent, ordering is done alphabetically. 
+
+|       |         |
+| ----- | ------- |
+| ![purple cube for a method](images/intellisense/method_icon.png) | Method or Function |
+| ![blue cuboid for a variable](images/intellisense/variable_icon.png) | Variable | 
+| ![blue circles connected by a blue line for an object](images/intellisense/object_icon.png) | Object |
+| ![a square with a small fold in the top left corner indicates word completion](images/intellisense/word_completion_icon.png) | Word Completion |
+| ![a square with a small fold in the top left corner indicates string completion](images/intellisense/string_completion_icon.png) | String completion |
+
+## Customize IntelliSense
+
+You can customize your IntelliSense experience in settings. 
+
+> Tip: The settings shown below are the default settings. You can change these settings in your settings.json file as described [here](/docs/customization/userandworkspace.md). 
+
+```json
+{
+
+    // Controls if quick suggestions should show up while typing
+    "editor.quickSuggestions": true,
+
+    // Controls if suggestions should be accetped with "Enter" - in addition to "Tab". Helps to avoid ambiguity between inserting new lines and accepting suggestions. 
+    "editor.acceptSuggestionOnEnter": true,
+
+    // Controls the delay in ms after which quick suggestions will show up. 
+    "editor.quickSuggestionsDelay": 10
+}
+```
+
+## How does IntelliSense work? 
```

Comment:
Do user's know they have this option? The topic is small so I don't think it hurts. Do you?